docs/shipping/Code/dotnet.md

```diff
@@ -32,70 +36,38 @@ import TabItem from '@theme/TabItem';
 * .NET Core SDK version 2.0 or higher
 * .NET Framework version 4.6.1 or higher
 
-:::note
-[Project's GitHub repo](https://github.com/logzio/logzio-dotnet/)
-:::
 
 
-#### Add the dependency to your project
 
-If you're on Windows, navigate to your project's folder in the command line, and run this command to install the dependency.
+### Add the dependency to your project
+
+On Windows, open the command line, go to your project folder, and run this command to install the dependency.
 
 ```
 Install-Package Logzio.DotNet.Log4net
 ```
 
-If you're on a Mac or Linux machine, you can install the package using Visual Studio. Select **Project > Add NuGet Packages...**, and then search for `Logzio.DotNet.Log4net`.
+On Mac or Linux, install the package using Visual Studio. **Project > Add NuGet Packages...**, and search for `Logzio.DotNet.Log4net`.
 
-#### Configure the appender
 
-You can configure the appender in a configuration file or directly in the code.
-Use the samples in the code blocks below as a starting point, and replace them with a configuration that matches your needs. See [log4net documentation 🔗](https://github.com/apache/logging-log4net) to learn more about configuration options.
+### Configure the appender in a configuration file
 
-For a complete list of options, see the configuration parameters below the code blocks.👇
-
-##### Option 1: In a configuration file
+Use the samples below as a starting point and adjust them to fit your needs. For more options, see the [log4net documentation](https://github.com/apache/logging-log4net). For a complete list of options, see the configuration parameters listed below.
 
 ```xml
 <log4net>
     <appender name="LogzioAppender" type="Logzio.DotNet.Log4net.LogzioAppender, Logzio.DotNet.Log4net">
-    	<!--
-		Required fields
-	-->
-	<!-- Your Logz.io log shipping token -->
 	<token><<LOG-SHIPPING-TOKEN>></token>
-
-	<!--
-		Optional fields (with their default values)
-	-->
-	<!-- The type field will be added to each log message, making it
-	easier for you to differ between different types of logs. -->
     	<type>log4net</type>
-	<!-- The URL of the Logz.io listener -->
     	<listenerUrl>https://<<LISTENER-HOST>>:8071</listenerUrl>
-        <!--Optional proxy server address:
-        proxyAddress = "http://your.proxy.com:port" -->
```

Comment:
this for example should probably not be deleted, right? 
who did you consult with?

Reply:
generally you should remove lines if:
1. the info is written somewhere else, and this is a duplication.
2. the info is not relevant
3. the name of the tag is self explanatory - for example, here the comment do not add any further information to the tag name: 
`<!-- The URL of the Logz.io listener -->
    	<listenerUrl>https://<<LISTENER-HOST>>:8071</listenerUrl>`